src/index.html

```diff
@@ -1,13 +1,346 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link
+      rel="shortcut icon"
+      href="images/icons/favicon.png"
+      type="image/x-icon"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body" id="top-page">
+    <!-- #region header -->
+    <header class="header page__header">
+      <div class="header__container">
+        <div class="top-bar page__top-bar ">
+          <a href="#top-page" class="top-bar__link">
+            <img
+              src="images/logo-white-small.svg"
+              alt="MyBike logo"
+              class="top-bar__logo"
+            >
+          </a>
+
+          <div class="top-bar__icons">
+            <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
```

Comment:
https://github.com/mate-academy/layout_landing-page/assets/106310796/c56764bf-e521-4b30-9800-8854601969ce

Reply:
transition should work both ways